### PR TITLE
added extra check to file object

### PR DESCRIPTION
**Pull request by jeanawhou**
@Quiescent @cameronsutter i noticed this error while testing a fix for a bug from the airtable

Bug:

https://user-images.githubusercontent.com/19387007/173503792-1db1356c-593c-4d1c-b4f0-41bcc6c69fb4.mp4

Fix demo

https://user-images.githubusercontent.com/19387007/173501378-8b19391e-7493-4e04-885d-3e5d28fc96ba.mp4

### CODE REVIEW

File: lib/pltr/v2/migrator/handleSpecialCases.js [hunk 1]
```diff
@@ -41,9 +41,10 @@ export const handleMissingTemplatesFieldOnCards = (file) => {
 
   // Check, and leave the file completely un changed if there were no
   // problems, because that feels safer.
-  const thereAreMissingTemplates = file.cards.some(
-    (card) => !card.templates || !Array.isArray(card.templates)
-  )
+  const thereAreMissingTemplates =
+    file &&
+    file.card &&
```
Comment (Quiescent) [inline lib/pltr/v2/migrator/handleSpecialCases.js]: This looks like a typo(?)

Comment (Quiescent) [reply lib/pltr/v2/migrator/handleSpecialCases.js]: I think this caused the failing test.

**Comment by Quiescent** [comment]
I'm not sure what goes wrong in the demo.  Could you please elaborate?

File: lib/pltr/v2/migrator/handleSpecialCases.js (hunk 1 above)
Comment (jeanawhou) [reply lib/pltr/v2/migrator/handleSpecialCases.js]: Oh yes of course 😅 thank you for checking ed

**Comment by jeanawhou** [comment]
> I'm not sure what goes wrong in the demo. Could you please elaborate?

@Quiescent i updated the description 😅 sorry about it 🙇

**Review by Quiescent — APPROVED** [review]
Great work.  LGTM! :)

**Pull request merged**